src/pages/landing/weekend-trading/components/_icon-text-row.js

```diff
@@ -149,18 +151,25 @@ const IconTextRow = () => {
                         <img src={SettlePayment} alt="" width="95px" />
                     </Row>
                     <Row>
-                        <Column></Column>
                         <Column>
                             <StyledCardHeader
+                                pl="0.6rem"
+                                pr="0.6rem"
                                 mt="6.5rem"
-                                mb="0.8rem"
+                                mb="1.8rem"
```

Comment:
[Suggestion] Writing `p="0 0.6rem"` shorten the amount of code you need to write across the many components, including the media queries as well. Same for margin.

Reply:
done